src/node/entities.h

```diff
@@ -60,50 +60,68 @@ namespace ccf
+      "public:ccf.internal.members.digests";
+
+    // User identities
     static constexpr auto USERS = "public:ccf.gov.users";
-    static constexpr auto USER_CERT_DERS = "public:ccf.gov.user_cert_ders";
-    static constexpr auto USER_DIGESTS = "public:ccf.gov.user_digests";
+    static constexpr auto USER_CERT_DERS =
+      "public:ccf.internal.users.certs_der";
+    static constexpr auto USER_DIGESTS = "public:internal.gov.users.digests";
     static constexpr auto SERVICE_PRINCIPALS =
       "public:ccf.gov.service_principals";
+
+    // Nodes identities and allowed code ids
     static constexpr auto NODES = "public:ccf.gov.nodes";
-    static constexpr auto VALUES = "public:ccf.gov.values";
-    static constexpr auto CONSENSUS = "public:ccf.gov.consensus";
-    static constexpr auto WHITELISTS = "public:ccf.gov.whitelists";
+    static constexpr auto NODE_CODE_IDS = "public:ccf.gov.nodes.code_ids";
+
+    // Service information
+    static constexpr auto SERVICE = "public:ccf.gov.service";
+    static constexpr auto CONFIGURATION = "public:ccf.gov.service.config";
+
+    // Governance
     static constexpr auto PROPOSALS = "public:ccf.gov.proposals";
     static constexpr auto GOV_SCRIPTS = "public:ccf.gov.governance.scripts";
-    static constexpr auto APP_SCRIPTS = "public:ccf.gov.app_scripts";
-    static constexpr auto MODULES = "public:ccf.gov.modules";
-    static constexpr auto SECRETS = "public:ccf.gov.secrets";
-    static constexpr auto NODE_CODE_IDS = "public:ccf.gov.nodes.code_ids";
     static constexpr auto GOV_HISTORY = "public:ccf.gov.governance.history";
-    static constexpr auto SERVICE = "public:ccf.gov.service";
-    static constexpr auto SHARES = "public:ccf.gov.shares";
-    static constexpr auto CONFIGURATION = "public:ccf.gov.config";
-    static constexpr auto SUBMITTED_SHARES = "public:ccf.gov.submitted_shares";
-    static constexpr auto SNAPSHOT_EVIDENCE =
-      "public:ccf.gov.snapshot_evidence";
-    static constexpr auto CA_CERT_DERS = "public:ccf.gov.ca_cert_ders";
-    static constexpr auto JWT_ISSUERS = "public:ccf.gov.jwt_issuers";
+    static constexpr auto WHITELISTS = "public:ccf.gov.whitelists";
+
+    // JS applications
+    static constexpr auto APP_SCRIPTS = "public:ccf.app.scripts";
+    static constexpr auto MODULES = "public:ccf.app.modules";
+    static constexpr auto ENDPOINTS = "public:ccf.app.endpoints";
+
+    // JWT issuers
+    static constexpr auto CA_CERT_DERS = "public:ccf.gov.jwt.ca_certs_der";
+    static constexpr auto JWT_ISSUERS = "public:ccf.gov.jwt.issuers";
     static constexpr auto JWT_PUBLIC_SIGNING_KEYS =
-      "public:ccf.gov.jwt_public_signing_keys";
+      "public:ccf.gov.jwt.public_signing_keys";
     static constexpr auto JWT_PUBLIC_SIGNING_KEY_ISSUER =
-      "public:ccf.gov.jwt_public_signing_key_issuer";
-    static constexpr auto ENDPOINTS = "public:ccf.gov.endpoints";
+      "public:ccf.gov.jwt.public_signing_key_issuer";
```

Comment:
Another general discussion point, without an actionable recommendation yet:

The `JWT` and `users` tables don't need to be in every service. There's a combination of proposals and tables and auth policies which all apps _may_ use, and since they're framework-standard I think its right that they get `ccf.gov.` prefixes. But if an app purely uses its own auth policies, ideally it should be able to opt out of these entirely. The maps are fine - they're simply never written to so never appear in the ledger. But we should think about hiding the proposals too, and encapsulating all of these definitions in a more obviously optional package.

Reply:
Unless the governance decides to use them, the overhead is nil. If it was just tables + proposals, we could make them pure JS after the conversion, but there's the polling job which needs to be done natively, so it's likely to stay that way in the foreseeable future.